src/array/convert_ndarray.rs

```diff
@@ -3,9 +3,33 @@ use crate::array::*;
 // ndarray has separate definitions of Array0 - Array6 although they are based on some generics.
 // We also provide separate definitions of TryFrom<&Array> for each ArrayN to fit the same manner.
```

Comment:
We might want to update the comment because the `define_into_array!` was added here.

```suggestion
// We also provide separate definitions of TryFrom<&Array> and IntoArray for each ArrayN to fit the same manner.
```